Src/Common/W3C/W3CActivityExtensions.cs

```diff
@@ -0,0 +1,210 @@
+﻿namespace Microsoft.ApplicationInsights.W3C
+{
+    using System;
+    using System.ComponentModel;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.Linq;
+
+    /// <summary>
+    /// Extends Activity to support W3C distributed tracing standard.
+    /// </summary>
+    [Obsolete("Not ready for public consumption.")]
```

Comment:
will this Obsolete attribute apply to extension methods inside? I just don't know how it works for extensions...